src/mover_manager.erl

```diff
@@ -210,8 +210,9 @@ ready({start, NumObjects, NumWorkers, Worker}, _From, CurrentState) ->
                    worker = #migration_worker{callback_module = CallbackMod } = Worker},
     call_if_exported(CallbackMod, migration_init, [], fun no_op/0),
     {reply, {ok, burning_couches}, working, State, 0};
-ready(create_account_dets, _From, State = #state{ acct_info = Acct} ) ->
-    NewAccount = create_dets_files(Acct),
+ready(create_account_dets, _From, State = #state{ acct_info = _Acct} ) ->
+    %%NewAccount = create_dets_files(Acct),
```

Comment:
Calling this out as the hard dep on couch we need to make configurable or whatever.